### DISCUSSION TITLE
I am leaving GitHub.

### DISCUSSION THREAD

**Discussion by hexaheximal** [community/community#63943] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Platform Abandonment

### Body

Here are some sources for why GitHub is harmful to the development ecosystem:

- https://nogithub.codeberg.page/
- https://librecmc.org/github.html
- https://sfconservancy.org/GiveUpGitHub/

As I've described [here](https://github.com/orgs/community/discussions/63813), I am not happy with GitHub's recent decision to make 2FA mandatory. And that has finally pushed me to reconsider GitHub.

So... I'm now migrating to codeberg: https://codeberg.org/hexaheximal

**Comment by ghost** [comment]
I am going to Gitlab.com. Private projects already use standalone on-premise instance. So I have big experience in that.

_Please wait, Gihub is shutting down..._

)))

**Reply by hexaheximal** [reply]
GitLab is still proprietary and for-profit. I recommend Codeberg instead - that's what I'm moving to.
